class/app-storage.js

```diff
@@ -269,6 +269,21 @@ export class AppStorage {
     this.wallets = [...this.wallets, w];
   }
 
+  updateWallet(w) {
```

Comment:
what is 'w'? Please, consider rewriting it to ts

Reply:
w is the wallet I can rename it to walletUpdate. There is a  separate task for rewriting app-storage to ts